utils/decorators.py

```diff
@@ -0,0 +1,3 @@
+def notCallable(cls):
+    cls.do_not_call_in_templates = True
+    return cls
```

Comment:
Is this decorator necessary?

Reply:
Yes, it is absolutely necessary in order to make enums accessible in templates. Otherwise no value is displayed after render.
I forgot what was exactly the issue, but it has something to do with calls in Django.